slices/xz-utils.yaml

```diff
@@ -0,0 +1,36 @@
+package: xz-utils
+
+essential:
+  - xz-utils_copyright
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - liblzma5_libs
+    contents:
+      /usr/bin/lzmainfo:
+      /usr/bin/unxz:
+      /usr/bin/xz:
+      /usr/bin/xzcat:
+  utils:
+    essential:
+      - base-files_bin
+      - base-files_tmp
+      - coreutils_bins
+      - dash_bins
```

Comment:
I disagree with adding dash as a dependency if not explicitly mentioned by the package itself. Dash is only one of the possible shells, and bash would also work. I don't think we should get to decide this

Reply:
Let me fix that now that we are not going with any pre-defined shell. :)